src/main/java/james/altoclef/taskcreator/graphics/AltoFrame.java

```diff
@@ -416,6 +433,14 @@ private void exploreForJson(FileDialog fd) throws IOException, ParseException {
         refreshTable();
     }
 
+    private void loadFile(String path) throws IOException, ParseException {
+        manager = new JSONManager(path);
+        file = manager.getFile();
+        setTitle("Altoclef-TaskCreator -- " + manager.getFileName());
+        l_shareableString.setText("");
+        refreshTable();
+    }
+
```

Comment:
why can we not use the function below this? Or was this moved from somewhere? I feel like we could refactor this and the below function as to have only a single `loadFile` function to simplify the `AltoFrame` class.